README.md

```diff
@@ -192,6 +192,20 @@ charts:
         buildArgs:
           MY_STATIC_BUILD_ARG: "hello world"
           MY_DYNAMIC_BUILD_ARG: "{TAG}-{LAST_COMMIT}"
+        # Build options to be passed to the docker build command. Pass a list
+        # of strings to be appended to the end of the build command. These are
+        # passed directly to the command line, so prepend each option with "--"
+        # like in the examples below. TAG and LAST_COMMIT are expandable.
+        extraBuildCommandOptions:
+          - --label=maintainer=octocat
+          - --label=ref={TAG}-{LAST_COMMIT}
+          - --rm
+          # above is equivalent to
+          - --label
+          - maintainer=octocat
+          - --label
+          - ref={TAG}-{LAST_COMMIT}
+          - --rm
```

Comment:
Now that we have pure string appending without also prefixing the strings with `--`, this is easy to understand in general I think, so I'll remove this part that now feels redundant.

```suggestion
```